src/Microsoft.Docs.Build.Specialized/views/Template/BuildItemListPartial.cshtml

```diff
@@ -36,7 +36,7 @@
     }
 }
 
-@if (@Model.Style == "unordered")
+@if (@Model?.Style == "unordered")
```

Comment:
🙄

Reply:
yes, there are new `null` annotations that give us new warnings, which is good.